apps/air-discount-scheme/api/infra/api.ts

```diff
@@ -36,6 +36,7 @@ export const serviceSetup = (services: {
         staging: 'cdn.contentful.com',
         prod: 'cdn.contentful.com',
       },
+      NO_UPDATE_NOTIFIER: 'true',
```

Comment:
Maybe set this in the global env section?

Reply:
Are you talking about file in `infra/src/environments.ts`? It is already set there but it did not work in every location which is really strange

Reply:
That's weird. Let's fix that issue after this fix.